### ISSUE TITLE
Sign In takes me to Sentry instead of signing me into the docs

### ISSUE THREAD

**Issue by bruno-garcia** [getsentry/sentry-docs#11557]
### Steps to Reproduce

![Image](https://github.com/user-attachments/assets/f2b3837b-bcc4-40a9-b89b-77d83322178a)

When I hit Sign In, I end up in Sentry looking at my event stream

### Expected Result

My Name would show at the top right, and I'd see proper DSN's on the docs instead of templates

### Actual Result

Navigated away from the docs, into Sentry

**Comment by stephanie-anderson** [comment]
This is the desired behaviour (being redirected to the product, and not being redirected back to the platform on which you clicked `Sign In`). See changelog etc

**Comment by whitep4nth3r** [comment]
The DSN population is _super useful_ and would be move obvious if there was an obvious "signed in" state in the nav or sidebar.

On a number of occasions, I have clicked "Sign in" and have been surprised each time it took me to Sentry.

As @bruno-garcia suggests, perhaps the "Sign in" text should be changed to "Go to Sentry" or similar.

**Comment by rahulchhabria** [comment]
i'm good with "go to sentry" but i find it super interesting that you both want to sign into docs. do other docs sites work that way?

**Comment by whitep4nth3r** [comment]
> do other docs sites work that way?

Yes, indeed!

Airtable is a great example — if you're signed in, the code samples come pre-populated with your own data — which is what we do on the Sentry docs. Stripe do this, too. And I'm sure many others.

**Comment by stephanie-anderson** [comment]
Then let's find some other wording for `Sign In` as long as we want users to be redirected to the product afterwards. 

Having a logged in state in docs is another issue though, and something that we have experimented with in the past (see https://github.com/getsentry/sentry-docs/pull/11124). The UX was really bad though, so we reverted the change (it wasn't easily possible to test this locally or on Vercel's preview environment).

For it to be a really great experience, we would need to
- be able to test this properly
- make the page dynamic (instead of statically generated files). Otherwise the right top section would always flicker with each new page request, which was not a solution we could live with

Having said that - having a logged in state was something that we came up with ourselves (docs team + docs platform engineers), but haven't heard much feedback from others. So we didn't see justification for spending the engineering effort into making it possible after this initial setback.

Will discuss this issue in our upcoming sync on Monday and revive the discussion. But for this particular `Sign In` button, which should lead to Sentry (as it does on all the pages), we should simply change the wording imo. And change it on all the pages, not only docs
